@here/harp-map-theme/scripts/prepack.ts

```diff
@@ -21,3 +21,7 @@ function onCopyError(err: Error) {
 const fontDir = path.dirname(require.resolve("@here/harp-font-resources/package.json"));
 
 ncp(path.join(fontDir, "resources"), path.join(__dirname, "..", "resources", "fonts"), onCopyError);
+
```

Comment:
Why this change is needed these ? 

Shouldn't all configs files like `maki-day-11.json` be here in this package ?

Reply:
We may do it like that, I just saw that we are coping font resources from already existing package so done similar way (out harp-atlas-tools already have configs as examples). What do you suggest commit those configs to this package? If so, please tell me resources-dev/ is it correct path, or should I create some other folder?

Reply:
> we are coping font resources from already existing 

Yes, but this is only because font-resources are large and no-one wants to rebuild them on each built.

> out harp-atlas-tools already have configs as examples)

But they are examples, guys modyfying atlas tools and playing with examples shouldn't affect configs and assets for this particular use case - maki-based icons for `berlin` themes ...

Reply:
Ok, fine for me too.

Reply:
Done.